streams/piping/general.js

```diff
@@ -154,4 +154,37 @@ promise_test(() => {
 
 }, 'Piping from a ReadableStream for which a chunk becomes asynchronously readable after the pipeTo');
 
+for (const preventAbort of [true, false]) {
+  promise_test(t => {
+
+    const rs = new ReadableStream({
+      pull() {
+        return Promise.reject(undefined);
+      }
+    }, { preventAbort });
+
+    return promise_rejects(t, null, rs.pipeTo(new WritableStream()), 'promise should be rejected');
```

Comment:
Hmm. I think this will allow anything through with no `name` property? Until https://github.com/w3c/testharness.js/issues/165 is landed this might be a good case for actually reverting to the assert_unreached + assert_equals combo to test that the rejection reason is actually undefined.